packages/pn-pa-webapp/src/pages/components/Notifications/FilterNotifications.tsx

```diff
@@ -69,6 +69,33 @@ function isFilterapplied(filtersCount: number): boolean {
 
 const getValidStatus = (status: string) => (status === 'All' ? '' : status);
 
+// the formik validations (which control the enable status of the "filtra" button)
+// must coincide with the input field validations (which control the color of the frame around each field)
+const dateFieldsValidation = {
+  // startDate cannot be earlier than 10 years ago
+  startDate: yup.date().min(tenYearsAgo).test({
+    name: 'notInFuture-start',
+    test(value) {
+      return !value || value.getTime() <= today.getTime();
+    }
+  }),
+  // endDate cannot be earlier than 10 years ago
+  endDate: yup.date().min(tenYearsAgo).test({
+    name: 'notInFuture-end',
+    test(value) {
+      return !value || value.getTime() <= today.getTime();
+    }
+  // endDate cannot be earlier than start date
+  }).test({
```

Comment:
We can use the min condition. In the component we have two state variables startDate and endDate.
So we can do something like that 

```
endDate: yup
      .date()
      .min(startDate && !isNaN(startDate.getTime()) ? startDate : tenYearsAgo)
```
This code increase the complexity, so we can define a function

`
const dateIsDefined = (date: Date | null) => date && !isNaN(date.getTime());
`

in date.utility.ts

Reply:
ok done as suggested